templates/collaboration.html

```diff
@@ -0,0 +1,180 @@
+<link href="/styles/fontawesome/css/all.css" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="/styles/app.css">
+<style type="text/css">
+*{
+	margin: 0px;
+	padding:0px;
+}
+
+.input{
+	flex:1;
+	border:1px solid rgba(36,41,46,.4);
+	border-radius: 3px;
+	border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+}
+
+.side-bar{
+	width: 30%;
+	display: flex;
+	flex-direction: column;
+	border: 1px solid #ddd;
+	border-radius: 3px;
+	margin-right: 10px;
+}
+.section{
+	margin: 50px;
+	margin-right:200px;
+	margin-left:200px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.main-section{
+	width:70%;
+	border: 1px solid #ddd;
+	border-radius: 3px;
+}
+.side-keys{
+	padding: 20px;
+	border-bottom: 1px solid #ddd;
+}
+.side-keys a {
+	text-decoration:none;
+	color: black;
+}
+.heading{
+	padding: 10px;
+	border-bottom: 1px solid #ddd;
+	background-color: #f6f8fa;
+	display: flex;
+	justify-content: space-between;
+}
+.info{
+	padding: 30px;
+	border-bottom: 1px solid #ddd;
+}
+.button{
+	padding: 10px;
+	border:1px solid rgba(36,41,46,.4);
+	border-radius: 3px;
+	background-color: #eff3f6;
+	min-width: 40px;
+	color: rgba(36,41,46,.4);
+	font-weight: bold;
+	border-left: 0;
+	border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+}
+.collaborator-section{
+	margin: 10px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.space{
+	padding: 10px;
+}
+.user-photo{
+	padding-right: 10px;
+	width: 50px;
+	height: 50px;
+}
+.collaborator{
+	display: flex;
+	justify-content: space-between;
+	padding: 10px;
+	padding-left: 20px;
+	border-bottom: 1px solid #ddd;
+}
+.collaborator-info{
+	padding: 40px;
+	border-bottom: 1px solid #ddd;
+	text-align: center;
+}
+.search-text{
+	padding: 10px;
+	font-weight: bold;
+	font-size: 20px;
+}
+.collaborator .username a{
+	text-decoration: none;
+	color: #666;
+	font-weight: bold;
+	font-size: 18px;
+	padding-bottom: 10px;
+}
+
+.search{
+	background-color: #fcfcfc;
+}
+
+.collaborator .username div{
+	font-size: 12px;
+	padding-top: 10px;
+}
+
+.close-button{
+	border :0;
+	padding: 0;
+	text-align:end;
+}
+
+.collaborator-details{
+	display: flex;
+}
+
+
+</style>
+{{template "nav.html" .}}
+{{if .Authorized}}
```

Comment:
No need for two if statements. We can use `and` function like `eq`. Write 

```
{{ if and .Authorized .IsRepoOwner }}
     ....
{{ end }}
````

Reply:
Done